libminifi/test/TestBase.cpp

```diff
@@ -234,12 +234,16 @@ std::shared_ptr<minifi::core::Processor> TestPlan::addProcessor(const std::share
   processor->setFlowIdentifier(flow_version_->getFlowIdentifier());
   processor_mapping_[processor->getUUID()] = processor;
   if (!linkToPrevious) {
-    termination_ = *(relationships.begin());
+    if (!std::empty(relationships)) {
+      termination_ = *(relationships.begin());
```

Comment:
```suggestion
      termination_ = relationships.front();
```

Reply:
It seems to be breaking the build. I thought this is a vector, but the error msg says it's an `std::initializer_list`.

Reply:
reverted